textattack/commands/attack/attack_args_helpers.py

```diff
@@ -414,28 +414,54 @@ def parse_dataset_from_args(args):
 def parse_logger_from_args(args):
     # Create logger
     attack_log_manager = textattack.loggers.AttackLogManager()
-    out_time = int(time.time() * 1000)  # Output file
-    # Set default output directory to `textattack/outputs`.
-    if not args.out_dir:
-        current_dir = os.path.dirname(os.path.realpath(__file__))
-        outputs_dir = os.path.join(
-            current_dir, os.pardir, os.pardir, os.pardir, "outputs", "attacks"
-        )
-        if not os.path.exists(outputs_dir):
-            os.makedirs(outputs_dir)
-        args.out_dir = os.path.normpath(outputs_dir)
+
+    # Get current time for file naming
+    year, month, day, hour, minute = map(str, time.strftime("%Y %m %d %H %M").split())
+    timestamp = f"{year}-{month}-{day}-{hour}-{minute}"
+
+    # Get default directory to save results
+    current_dir = os.path.dirname(os.path.realpath(__file__))
+    outputs_dir = os.path.join(
+        current_dir, os.pardir, os.pardir, os.pardir, "outputs", "attacks"
+    )
+    if not os.path.exists(outputs_dir):
+        os.makedirs(outputs_dir)
+    out_dir_txt = out_dir_csv = os.path.normpath(outputs_dir)
+
+    # Get default txt and csv file names
+    if args.recipe:
+        filename_txt = f"{args.model}_{args.recipe}_{timestamp}.txt"
+        filename_csv = f"{args.model}_{args.recipe}_{timestamp}.csv"
+    else:
+        filename_txt = f"{args.model}-{timestamp}.txt"
+        filename_csv = f"{args.model}-{timestamp}.csv"
```

Comment:
What do you think if we make the two namings here consistent (choosing either between dash `-` and underscore `_`)?

Reply:
Thank you! Didn't notice that earlier...